tests/acceptance/cli_test.py

```diff
@@ -718,6 +718,15 @@ def test_up_handles_force_shutdown(self):
         os.kill(proc.pid, signal.SIGTERM)
         wait_on_condition(ContainerCountCondition(self.project, 0))
 
+    @v2_only()
+    def test_up_handles_abort_on_container_exit(self):
+        self.base_dir = 'tests/fixtures/sleeps-composefile'
+        start_process(
+            self.base_dir,
+            ['-f', 'one-exits.yml', 'up', '-t', '200', '--abort-on-container-exit'])
```

Comment:
I think this `-t` timeout should be `-t 1`, The value is set for 200 just for that one test, because it's relevant in that case, but in this case we know that `sleep` requires a SIGKILL to shutdown, so we don't want to wait around for it.